docs/communication/fake_server.md

```diff
@@ -1,79 +0,0 @@
-FakeServer
```

Comment:
Why is the doc for the FakeServer deleted?

Reply:
It's moved to testing, where it belongs :-)

Reply:
https://github.com/qooxdoo/qooxdoo/pull/9890/files#diff-0ed61c13d9c4d30feeae53289d819841